lib/locales/en.ts

```diff
@@ -857,6 +878,8 @@ export const locale: Locale = {
     pontian_i_pope_and_hippolytus_of_rome_priest: 'Saints Pontian, Pope, and Hippolytus, Priest, Martyrs',
     pothinus_of_lyon_bishop_blandina_of_lyon_virgin_and_companions_martyrs:
       'Saints Pothinus, Bishop, Blandina, Virgin, and Companions, Martyrs',
+    pothinus_of_lyon_bishop_patron_the_city_of_lyon_blandina_of_lyon_virgin_and_companions_martyrs:
+      'Saints Pothinus, Bishop, Patron of the city of Lyon, Blandina, Virgin, and Companions, Martyrs',
```

Comment:
In English, word `city`/`town` is always written with captital initial letter if it is used before a specific city/town.

```suggestion
    pothinus_of_lyon_bishop_patron_the_city_of_lyon_blandina_of_lyon_virgin_and_companions_martyrs:
      'Saints Pothinus, Bishop, Patron of the City of Lyon, Blandina, Virgin, and Companions, Martyrs',
```